zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -81,17 +97,26 @@ def _update_attribute(self, attrid, value):
                 self.TEMPERATURE_ATTRS[attrid],
                 value * 10,
             )
-        elif attrid == SASSWEL_STATE_ATTR:
+        elif attrid == SASWELL_STATE_ATTR:
             self.endpoint.device.thermostat_bus.listener_event(
                 "system_mode_reported", value
             )
-        elif attrid == SASSWEL_BATTERY_LOW_ATTR:
+        elif attrid == SASWELL_BATTERY_LOW_ATTR:
             # this device doesn't have battery level reporting, only battery low alert
             # when the alert is active (1) we report 0% and 100% otherwise (0)
             self.endpoint.device.battery_bus.listener_event(
                 "battery_change", 0 if value == 1 else 100
             )
 
+        if attrid == (SASWELL_TEMPERATURE_CALIBRATION_ATTR):
+            self.endpoint.device.temperature_calibration_bus.listener_event(
+                "set_value", value
+            )
+        elif attrid in (SASWELL_LOCAL_TEMP_ATTR, SASWELL_HEATING_SETPOINT_ATTR):
+            self.endpoint.device.thermostat_bus.listener_event(
+                "state_temp_change", attrid, value
+            )
```

Comment:
What is the intention of this? Where it come from?

The `thermostat_bus` doesn't have an `state_temp_change` event listener:
* https://github.com/zigpy/zha-device-handlers/blob/c2b2893d027fdcd3a225996fdf382d2ac72ff799/zhaquirks/tuya/__init__.py#L594

Reply:
No idea, I just copied the stuff over from the file you suggested until I got something working. I don't really understand all the building blocks yet. Is that not needed?

Reply:
Yeah guess it doesn't do anything really. I removed it.